src/test/java/com/iota/iri/conf/ConfigTest.java

```diff
@@ -199,6 +212,13 @@ public void testIniParsingMainnet() throws Exception {
         Assert.assertEquals("PORT", 17000, iotaConfig.getPort());
         Assert.assertEquals("NEIGHBORS", Arrays.asList("udp://neighbor1", "neighbor", "tcp://neighbor2"),
                 iotaConfig.getNeighbors());
+
+        List<InetAddress> expectedTrustedApiHosts = new ArrayList<>();
+        expectedTrustedApiHosts.add(InetAddress.getByName("192.168.0.55"));
+        expectedTrustedApiHosts.add(InetAddress.getByName("10.0.0.10"));
+        expectedTrustedApiHosts.add(BaseIotaConfig.Defaults.REMOTE_LIMIT_API_DEFAULT_HOST);
+        Assert.assertEquals("REMOTE_TRUSTED_API_HOSTS", expectedTrustedApiHosts, iotaConfig.getRemoteTrustedApiHosts());
+
```

Comment:
:+1: For adding unit tests :-)

I think you can maybe create the list in one line with `IotaUtils` or `Arrays.asList()`.
However, if you think your way is neater then you don't have to change :-)